src/CardPage.js

```diff
@@ -0,0 +1,210 @@
+import Card from './Card';
+
+export default class CardPage {
+  constructor() {
+    this.htmlPage = document.createElement('div');
+    this.htmlPage.classList.add('cards__page');
+    this.htmlPage.innerHTML = `
+            <div class="nav__bar">
+                <div class="star__bar"></div>
+                <button class="start__game__button">start</button>
+            </div>
+            <div class="cards__box"></div>
+        `;
+    this.startButton = this.htmlPage.querySelector('.start__game__button');
+    this.htmlStarBar = this.htmlPage.querySelector('.star__bar');
+    this.htmlCardBox = this.htmlPage.querySelector('.cards__box');
+    this.gameOverAudio = new Audio('assets/audio/game_over.mp3');
+    this.winAudio = new Audio('assets/audio/win.mp3');
+    this.wrongAudio = new Audio('assets/audio/wrong.mp3');
+    this.correctAudio = new Audio('assets/audio/correct.mp3');
+    this.htmlContainer = null;
+    this.correctCallback = null;
+    this.wrongCallback = null;
+    this.trainsCallback = null;
+    this.gameMode = false;
+    this.isHasWrong = false;
+    this.cards = [];
+    this.gameCards = [];
+
+    this.startButton.onclick = this.startGame.bind(this);
+  }
+
+  appendTo(parent) {
+    parent.appendChild(this.htmlPage);
+  }
+
+  createCards(data) {
+    data.words.forEach((word) => {
+      const card = new Card(word.eng, word.rus, data.category, word.pic, word.audio, data.color, data.subcolor);
+      card.appendTo(this.htmlCardBox);
+      this.cards.push(card);
+
+      if (this.gameMode) {
+        card.toggleMode();
+        card.setCallback(this.checkAnswer.bind(this));
+      } else {
+        card.setCallback(this.trainsCallback.bind(this));
+      }
+    });
+
+    if (this.htmlContainer) {
+      this.appendTo(this.htmlContainer);
+    }
+  }
+
+  reset() {
+    this.htmlStarBar.innerHTML = '';
+    this.isHasWrong = false;
+    this.gameCards = [];
+    this.startButton.textContent = 'start';
+  }
+
+  clearPage() {
+    this.htmlContainer.innerHTML = '';
+    this.htmlCardBox.innerHTML = '';
+    this.htmlStarBar.innerHTML = '';
+    this.cards = [];
+  }
+
+  toggleGameMode() {
+    this.gameMode = !this.gameMode;
+
+    if (this.gameMode) {
+      this.startButton.classList.add('start__game__button-show');
```

Comment:
If you use BEM class should be named as `start-game__button-show` or `start__game-button-show`